src/constants/deviceParameters.ts

```diff
@@ -2,6 +2,26 @@
  * Device parameters. Used in SetParameter/GetParameter.
  */
 
+/**
+ * Interval of data sending from device.
+ */
```

Comment:
as all these comments are rather simple what about one-liners?

```suggestion
/** interval of data sending from device */
```

Reply:
don't like this, but ok, fixed